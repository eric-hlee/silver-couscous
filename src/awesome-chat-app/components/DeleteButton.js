import React from 'react'

const style_button = {
  display: 'inline',
  fontSize: '12px',
  border: '1px solid #555555',
  position: 'absolute',
  float: 'right',
  boxShadow: '1px 2px',
  overflow: 'hidden',
  cursor: 'pointer',
}

const DeleteButton = ({ onClick }) => (
  <button onClick={onClick} style={style_button}>
    X
  </button>
)

export default DeleteButton