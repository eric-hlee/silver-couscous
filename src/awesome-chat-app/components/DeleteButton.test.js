import React from 'react'
import { shallow } from 'enzyme'
import DeleteButton from './DeleteButton'

describe('DeleteButton', () => {
  it('should render self and subcomponents', () => {
    const wrapper = shallow(<DeleteButton />)
    expect(wrapper).toMatchSnapshot()
  })
})
