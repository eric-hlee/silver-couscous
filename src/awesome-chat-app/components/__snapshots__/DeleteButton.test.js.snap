// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeleteButton should render self and subcomponents 1`] = `
<button
  style={
    Object {
      "border": "1px solid #555555",
      "boxShadow": "1px 2px",
      "cursor": "pointer",
      "display": "inline",
      "float": "right",
      "fontSize": "12px",
      "overflow": "hidden",
      "position": "absolute",
    }
  }
>
  X
</button>
`;
